src/components/Spinner/Spinner.js

```diff
@@ -1,62 +1,58 @@
-import React from 'react';
-import { View, Animated, Easing } from 'react-native';
+import React, { useEffect } from 'react';
+import { Animated, Easing, View } from 'react-native';
 import styled from '@stream-io/styled-components';
+
 import { themed } from '../../styles/theme';
 
 const AnimatedView = Animated.createAnimatedComponent(View);
 
 export const Circle = styled(AnimatedView)`
-  height: 30px;
-  width: 30px;
-  margin: 5px;
-  border-width: 2px;
-  border-radius: 30px;
-  justify-content: center;
   border-color: ${({ theme }) => theme.colors.primary};
+  border-radius: 30px;
   border-right-color: transparent;
+  border-width: 2px;
+  height: 30px;
+  justify-content: center;
+  margin: 5px;
+  width: 30px;
   ${({ theme }) => theme.spinner.css}
 `;
+
 /**
  * @example ../docs/Spinner.md
- * @extends PureComponent
  */
-class Spinner extends React.PureComponent {
-  static themePath = 'spinner';
-  state = {
-    rotateValue: new Animated.Value(0),
-  };
+const Spinner = () => {
+  const rotateValue = new Animated.Value(0);
```

Comment:
shouldn't this be a useRef otherwise the animation value will be reinitialized every rerender

Reply:
yeah looks like it works fine both ways, this component only shows up for a split second on initial ChannelList load and on paginating the list, so didn't think it needed to be ref'd up, but can make that change